### ISSUE TITLE
omex.initializeFromArchive(path) aborts python kernel

### ISSUE THREAD

**Issue by matthiaskoenig** [sbmlteam/libCombine#6]
When calling `initFromArchive` for files which do not exist or are no zip files the python kernel is aborted.
The c++ library should not kill the python kernel, but catch the runtime_error and return None.

``` Python
import libcombine

def validate_omex(path):
    """ Simple validation by trying to load. """
    omex = libcombine.CombineArchive()
    print("Init archive:")
    if not omex.initializeFromArchive(path):
        print("Invalid Combine Archive")


if __name__ == "__main__":
    path = "BIOMD0000000012.sedx1"
    validate_omex(path)
```

```
terminate called after throwing an instance of 'std::runtime_error'
  what():  Error loading zip file!
Aborted (core dumped)
```

**Comment by matthiaskoenig** [comment]
[BIOMD0000000003.sedx.zip](https://github.com/sbmlteam/libCombine/files/533780/BIOMD0000000003.sedx.zip)
Here an example file.

This makes it impossible to use libcombine because there is no way to check if a file is valid and it will just kill the process which tries to open the archive.

**Comment by fbergmann** [comment]
This should now be resolved with commit 7e2fe08d478eba7c87fb01655c8b2fd0f9d0f8a6, now all occuring c++ exceptions are caught in wrapped languages ... additionally initializeFromArchive now returns False on non-existing files as well.

**Comment by matthiaskoenig** [comment]
Is working. Thanks.
